charts/thoras/templates/api-server-v2/admission.yaml

```diff
@@ -18,7 +18,8 @@ metadata:
 data:
   {{- $secretObj := (lookup "v1" "Secret" .Release.Namespace "thoras-api-server-cert") }}
   {{- $secretData := (get $secretObj "data") | default dict }}
-  {{- $secretCert := (get $secretData "tls.crt") | default ($cert.Cert | b64enc) }}
+
+  {{- $secretCert := (get $secretData "tls.crt") | default (printf "%s\n%s" $cert.Cert $ca.Cert | b64enc) }}
```

Comment:
The v2 api needs to serve the whole cert chain for the dashboard and forecast to trust it

Reply:
lol wow -- how did this ever work if nothing's changing on the clients ends? Just for my own curiosity

Reply:
We're shippin!! 🚢